nvflare/fuel/f3/streaming/tools/README.md

```diff
@@ -0,0 +1,31 @@
+# Streaming API Testing Tools
+
+The following command line tools are provided to test the streaming API.
+
+There are 2 sets of tools:
+1. sender.py/receiver.py: Those can be used to send a memory buffer to the remote cell. The memory buffer size must
+   fit in the memory.
+2. file_sender.py/file_receiver.py: This uses StreamCell to send/receive files. This can be used to send 
```

Comment:
there is no file_sender.py/file_receiver.py anymore

Reply:
No. The API they use is removed.

Reply:
Then should we remove this "2. file_sender.py/file_receiver.py: This uses StreamCell to send/receive files. This can be used to send "?